resources/core/charts/console/values.yaml

```diff
@@ -17,7 +17,7 @@ core_ui:
   statusPort: 6081
   image:
     dir: 
-    tag: "477b7708"
+    tag: 5a8b31e2
```

Comment:
Shouldn't we also bump compass_mfs?

Reply:
We will have also labels display in applications today, so I'll wait for that. @Wawrzyn321 do you agree?